src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,92 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  onPageChange,
+  currentPage = 1,
+}) => {
+  const paginationLength = Math.ceil(total / perPage);
+  const paginationItems = Array(paginationLength).fill(0).map((_, i) => i + 1);
+  const prevIsDisabled = currentPage === 1;
```

Comment:
better for boolean use naming started with should, is, have, had

isPrevDisabled